templates/archive.php

```diff
@@ -1,18 +1,16 @@
-<header class="title col-md-9">
-    <hgroup>
-        <h1><?php echo h()->w_template_title(); ?></h1>
-        <?php if (category_description()) :?>
-            <h2><?php echo category_description(); ?></h2>
-        <?php endif; ?>
-    </hgroup>
+<header>
+    <h1><?php echo w_template_title(); ?></h1>
```

Comment:
Note that a few functions in `templates/` now need to be started with `h()->`, including this one.